recipes/doxyrest/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "2.0.0" %}
+
+package:
+  name: doxyrest
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/vovkos/doxyrest_b/releases/download/doxyrest-{{ version }}/doxyrest-{{ version }}-src.tar.xz
+  sha256: a7e93f0a3b45717440564c8e79929b7cfe39d164ba9b4a7f5ca0f88fc1f8747c
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - ragel
+    - expat
+    - lua
```

Comment:
I think ``ragel``, ``expat`` and ``lua`` should be in a ``host`` section. Could you please change it?

Reply:
Noted and fixed. However, I kept ragel in the build section -- it's a lexer compiler, so I believe it belongs in the build section together with cmake, gcc, etc.